com.microsoft.mrtk.standardassets/Materials/MRTK_Standard_LogoGradient.mat

```diff
@@ -0,0 +1,210 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
```

Comment:
Does the logo belong in StandardAssets? We've been trying to slim down and reduce the number of assets in SA. It should only be used in MRTK example projects, so, probably should only be in ExtendedAssets?

Reply:
Adjusted file location

Reply:
I don't think I agree. I guess I haven't looked at this specific logo, but we should definitely ship some MRTK logo variations in StandardAssets for using in MRTK editor windows and whatnot. It's a nice layer of polish that would be nice not to relegate to ExtendedAssets

Reply:
I guess this is a material though, so could be fine to treat a little differently than the image itself.

Reply:
+1 @cre8ivepark , probably best to put the material in Extended. We can still use the logo in Editor UI without the material.

Reply:
@Zee2 @keveleigh Yes, it is already moved to Extended assets. This comment thread shows an outdated location.
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/10751/files#diff-1dbcc1b259cb4afffd5864ca84c399e3f74cab4054d3ebe483a6f4c99edbca7f